preferences/fedora/x86_64/metadata/metadata.yaml

```diff
@@ -7,3 +7,5 @@ metadata:
     tags: "hidden,kubevirt,fedora"
     iconClass: "icon-fedora"
     openshift.io/display-name: "Fedora"
+  labels:
+    instancetype.kubevirt.io/arch: "x86_64"
```

Comment:
Should we add this to every preference?

Reply:
Yup, this is just a minimal draft only touching the Fedora preference.